.github/workflows/docs.yml

```diff
@@ -12,41 +12,19 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
-jobs:
-  linkcheck:
-    name: Sphinx linkcheck
-    runs-on: ubuntu-latest
-    env:
-      UV_SYSTEM_PYTHON: true
-    steps:
-      - uses: actions/checkout@v4
-      - name: Setup Python
-        uses: actions/setup-python@v5
-        with:
-          python-version: 3.13
-
-      - name: Set up uv
-        uses: astral-sh/setup-uv@v5
-        with:
-          cache-dependency-glob: |
-            blackbox/requirements.txt
-          enable-cache: true
-          version: "latest"
-
-      - name: Install dependencies
-        run: |
-          uv pip install --upgrade pip setuptools
-          uv pip install -r blackbox/requirements.txt
-
-      - name: Run linkcheck
-        run: |
-          sphinx-build -n -W --keep-going -q -c docs/ -b linkcheck -E docs/ docs/out/html
+permissions:
+  contents: read
+  checks: write
+  statuses: write
 
+jobs:
   vale:
     name: Vale
     runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v5
+        with:
+          persist-credentials: false
```

Comment:
why is this needed now? something new with `v5`?

Reply:
No not new, just ensures subsequent actions can't use the persisted credentials. See https://docs.zizmor.sh/audits/#artipacked